dist/ng-lazy-render.js

```diff
@@ -72,20 +77,35 @@ angular.module('ngLazyRender').directive('lazyRepeater', [
 
                 var bufferProp = tAttrs.ngRepeat.match(/in (.*?)?([ |\n|]|$)/)[1];
 
-                tElement.after('<div in-view="$inview && increaseLimit()"></div>');
-
                 return function ($scope, el, attrs) {
                     var limit = attrs.lazyRepeater;
                     var bufferLength = $scope.$eval(bufferProp).length;
+                    var template;
+                    var templateEl;
+                    var isolateScope;
 
-                    $scope.getLazyLimit = function () {
-                        return limit;
-                    };
+                    template = attrs.lazyTemplate ? $templateCache.get(attrs.lazyTemplate) || attrs.lazyTemplate : '';
```

Comment:
Can't we call the attr lazyRepeater? This way it doesn't look like another directive. And it matches the use-case of lazyModule="templateUrl".